internal/handlers/handlers.go

```diff
@@ -0,0 +1,74 @@
+package handlers
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/IgorGreusunset/shortener/cmd/config"
+	model "github.com/IgorGreusunset/shortener/internal/app"
+	"github.com/IgorGreusunset/shortener/internal/helpers"
+	"github.com/IgorGreusunset/shortener/internal/storage"
+	"github.com/go-chi/chi/v5"
+)
+
+func PostHandler(res http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
+		res.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
+	reqBody, err := io.ReadAll(req.Body)
```

Comment:
Не стоит забывать закрывать  `req.Body` после чтения.  

```suggestion
	reqBody, err := io.ReadAll(req.Body)
	defer req.Body.Close()
```